MIDIchlorians/MIDIchlorians/AnimationManager.swift

```diff
@@ -14,6 +14,13 @@ class AnimationManager {
     private var animationTypes = [String: AnimationType]()
 
     func getAllAnimationTypesNames() -> [String] {
+        let storedAnimationTypes = DataManager.instance.loadAllAnimationTypes()
+        _ = storedAnimationTypes.map({(animationTypeString: String) in
```

Comment:
Why use map if you aren't modifying the list at all? Wouldn't a normal for loop make more sense?